admin/google_search_console/class-gsc-service.php

```diff
@@ -119,9 +127,12 @@ public function fetch_category_issues( $platform, $category ) {
 			true
 		);
 
+		// Ignore coding standards for object properties.
+		// @codingStandardsIgnoreStart
 		if ( ! empty( $issues->urlCrawlErrorSample ) ) {
```

Comment:
Those properties should pass as long as they are on the whitelist as added in https://github.com/Yoast/wordpress-seo/pull/6875/commits/14f775fc7a7342dbf14e42a93870bf457b679d36